apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -142,6 +143,12 @@ const Courses = React.createClass({
             <ProtectedStatefulDiv ref="allCourses"/>
           </div>
         )}
+
+        {!isTeacher && (
+          <ProtectedStatefulDiv
```

Comment:
This would be better implemented as simply a button with a link entirely in React.  We only did this technique of taking existing DOM elements and copying them into the React DOM for components that were already implemented using HAML.